File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] orders) {
        int answer = 0;
        for(String order : orders){
            if(order.indexOf("americano") != -1)
                answer += 4500;
            else if(order.indexOf("cafelatte") != -1)
                answer += 5000;
            else
                answer += 4500;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> orders) {
    int answer = 0;
    for(auto order : orders){
        if(order.find("americano") != string::npos){
            answer += 4500;
        } else if(order.find("cafelatte") != string::npos){
            answer += 5000;
        } else{
            answer += 4500;
        }
    }
    return answer;
}